evol/population.py

```diff
@@ -214,6 +219,20 @@ def mutate(self, func, probability=1.0, **kwargs) -> 'Population':
             individual.mutate(func, probability=probability, **kwargs)
         return self
 
+    def log(self, **kwargs) -> 'Population':
+        """
+        Logs a population. If a Population object was initialized with a logger
+        object then you may specify how logging is handled. The base logging 
+        operation just logs to standard out. 
+        
+        :return: self
+        """
+        logs = self.logger.log(population=self, **kwargs)
```

Comment:
This feels a little redundant. Why do we need to call two methods of the logger, and doesn't it handle that by itself?